{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[tool.poetry]
+name = "{{ cookiecutter.project_name }}"
+version = "{{cookiecutter.version}}"
+description = "{{ cookiecutter.short_description }}"
+authors = ["{{ cookiecutter.author_name }} <{{ cookiecutter.author_email }}>"]
+homepage = "{{ cookiecutter.home_page }}"
+license = "BSD" # Replace with your project's license, if different
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    "Environment :: Web Environment",
+    "Framework :: Sphinx :: Extension",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Documentation",
+    "Topic :: Documentation :: Sphinx",
+    "Topic :: Utilities"
+]
+
+
+[tool.poetry.dependencies]
+python = "^3.6"
+pbr = "^5.5.1"
```

Comment:
tbh, pbr probably isn't the best horse to back now. Could we use setuptools here?

*Later* actually you us poetry so this should be wholly unnecessary?